color/src/colorspace.rs

```diff
@@ -353,6 +353,12 @@ impl ColorSpace for LinearSrgb {
     }
 }
 
+impl From<LinearSrgb> for ColorSpaceTag {
+    fn from(_: LinearSrgb) -> Self {
```

Comment:
In theory, this could be:
```rust
    fn from(LinearSrgb: LinearSrgb) -> Self {
```
which would make it robust against `LinearSrgb` gaining fields in the future (this would fail to compile in that case). However, I think that's sufficiently unlikely that we don't really need to worry about this.

That is, I don't think we should do this (but I wouldn't review against it if it were the state)